<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measures</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_distance">
    <attribute name="title">
     <string>Distance</string>
    </attribute>
    <widget class="QPushButton" name="button_plot1">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="edit_show1">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1"/>
    </widget>
    <widget class="QPushButton" name="button_clearplot1">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>360</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Plot</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>241</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="edit_label1">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="edit_atom1">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Atom selection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_add1">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_clear1">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_csv1">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Export csv</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_angle">
    <attribute name="title">
     <string>Angle</string>
    </attribute>
    <widget class="QPushButton" name="button_plot2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="edit_show2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
    <widget class="QPushButton" name="button_clearplot2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>360</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Plot</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>241</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLineEdit" name="edit_label2">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="edit_atom2">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Atom selection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_add2">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_clear2">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_csv2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Export csv</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_dihedral">
    <attribute name="title">
     <string>Dihedral</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
    <widget class="QPlainTextEdit" name="edit_show3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_plot3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_clearplot3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>360</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Plot</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>241</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLineEdit" name="edit_label3">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="edit_atom3">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Atom selection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_add3">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_clear3">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_csv3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Export csv</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_rmsd">
    <attribute name="title">
     <string>RMSD</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </widget>
    <widget class="QPlainTextEdit" name="edit_show4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_clearplot_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>360</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_plot4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>241</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLineEdit" name="edit_label4">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="edit_atom4">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Atom selection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_add4">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_clear4">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="button_csv4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>400</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Export csv</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
